src/CommandExecutor.ts

```diff
@@ -61,7 +67,8 @@ class SystemFileExecutionStrategy extends CommandExecutionStrategy {
 
             this.invocation.command.stdout.on('data', (data: string) => this.invocation.parser.parse(data.toString()));
             this.invocation.command.on('exit', (code: number) => {
-                if (code === 0) {
+                /* In windows there is no code returned (null) so instead of comparing to 0 we check if its 0 or null with ! */
+                if (!code) {
```

Comment:
How about converting to an integer first. The reason is only readability: `!code` doesn't look like a success case for me.